src/components/AvatarGroup/index.tsx

```diff
@@ -43,13 +43,7 @@ AvatarGroup.propTypes = {
      * This value defaults to medium. */
     size: PropTypes.oneOf(['large', 'medium', 'small', 'x-small']),
     /** An array with the Avatars data. */
-    avatars: PropTypes.arrayOf(
-        PropTypes.shape({
-            src: PropTypes.string,
-            initials: PropTypes.string,
-            icon: PropTypes.node,
-        }),
-    ),
+    avatars: PropTypes.array,
```

Comment:
set this back to:
```
avatars: PropTypes.arrayOf(
        PropTypes.shape({
            src: PropTypes.string,
            initials: PropTypes.string,
            icon: PropTypes.node,
        }),
    ),
```
it is needed for documentation, users need to know the shape of each avatar element